commercial/app/controllers/commercial/ContentApiOffersController.scala

```diff
@@ -123,17 +123,22 @@ class ContentApiOffersController(contentApiClient: ContentApiClient, capiAgent:
 
   private def renderNative(format: Format, isMulti: Boolean) = Action.async { implicit request =>
 
-    retrieveContent().map(_.toList.headOption).map(
-      {
-        case None => Cached(componentNilMaxAge){ jsonFormat.nilResult }
-        case Some(content) => {
-          val capiSingle = CapiSingle.fromContent(content)
-          Cached(60.seconds) {
-            JsonComponent(capiSingle)
-          }
+    retrieveContent().map {content =>
```

Comment:
IsMulti is a hangover from the previous methods. I suppose we could remove them now following all the changes